libnvme.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: libnvme
+  version: 1.10
+  epoch: 0
+  description: C Library for NVM Express on Linux
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - gcc
+      - json-c
+      - meson
+      - ninja
+      - openssl
+      - python3
+      - python3-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: dd51fa8550564c93436423a4d8ed4be92ae50290
+      repository: https://github.com/linux-nvme/libnvme
+      tag: v${{package.version}}
+
+  - uses: meson/configure
+
+  - uses: meson/compile
+
+  - uses: meson/install
+
+  - uses: strip
+
+subpackages:
+  - name: libnvme-dev
```

Comment:
In Wolfi, it's common practice that most `-dev` packages provides its actual package.

Reply:
thanks for the review, I addressed it via https://github.com/wolfi-dev/os/pull/32228/commits/1186c8ff4bbe2109b07bf4402eddd8983e5b1220